pandas/tslib.pyx

```diff
@@ -3530,7 +3530,7 @@ def tz_convert(ndarray[int64_t] vals, object tz1, object tz2):
     if _get_zone(tz2) == 'UTC':
         return utc_dates
 
-    result = np.empty(n, dtype=np.int64)
+    result = np.zeros(n, dtype=np.int64)
     if _is_tzlocal(tz2):
```

Comment:
i don't think it's supposed to actually be a string 
as _maybe_get_tz is called I think

but u can compare it to UTC